tests/test_belgium_calendar.py

```diff
@@ -0,0 +1,123 @@
+from unittest import TestCase
+import pandas as pd
+
+from .test_trading_calendar import ExchangeCalendarTestBase
+from trading_calendars.country_calendar_belgium import BelgiumCalendar
+
+
+class BelgiumCalendarTestCase(ExchangeCalendarTestBase, TestCase):
```

Comment:
I agree these could be consolidated - maybe have a `EuronextCalendarTestBase(ExchangeCalendarTestBase)` which contains test_normal_year and test_holidays_fall_on_weekend?